core/storage/feedback/gae_models_test.py

```diff
@@ -0,0 +1,124 @@
+        base_models.BaseModel.last_updated._auto_now = False
+        feedback_models.SuggestionModel.create('exp_id1', 'thread_id1',
+                                               'author_id', 1, 'state_name',
+                                               {'old_content': {}})
+        feedback_models.SuggestionModel.create('exp_id1', 'thread_id2',
+                                               'author_id', 1, 'state_name',
+                                               {'old_content': {}},
+                                               feedback_models.STATUS_ACCEPTED)
+        feedback_models.SuggestionModel.create('exp_id2', 'thread_id2',
+                                               'author_id', 1, 'state_name',
+                                               {'old_content': {}})
+
+    def get_suggestion_models_for_test(self, suggestions_list):
+        updated_suggestions_list = []
+        for suggestion in suggestions_list:
+            suggestion_dict = suggestion.to_dict()
+            if CREATED_ON_FIELD in suggestion_dict:
+                suggestion_dict.pop(CREATED_ON_FIELD)
+            updated_suggestions_list.append(suggestion_dict)
+        return updated_suggestions_list
+
+    def test_create_success(self):
+        suggestion = feedback_models.SuggestionModel.get_by_exp_id_and_status(
+            'exp_id1')[0]
+ 
+        self.assertEqual(suggestion.exploration_id, 'exp_id1')
+        self.assertEqual(suggestion.author_id, 'author_id')
+        self.assertEqual(suggestion.exploration_version, 1)
+        self.assertEqual(suggestion.state_name, 'state_name')
+        self.assertEqual(suggestion.state_content, {'old_content': {}})
+        self.assertEqual(suggestion.status, feedback_models.STATUS_NEW)
+
+    def test_create_failure(self):
+        with self.assertRaisesRegexp(Exception, 'There is already a feedback '
+                                     'thread with the given thread id: '
+                                     'exp_id1.thread_id1'):
+            feedback_models.SuggestionModel.create('exp_id1', 
+                                                   'thread_id1', 'author_id', 1, 
+                                                   'state_name', 
+                                                   {'old_content': {}})
+
+    def test_get_by_exp_id_and_status_status_set(self):
+        expected_suggestions = [feedback_models.SuggestionModel(
+            id='exp_id1.thread_id2',
+            author_id=u'author_id',
+            deleted=False,
+            exploration_id=u'exp_id1',
+            exploration_version=1,
+            state_name=u'state_name',
+            state_content={u'old_content': {}},
+            status=u'accepted')]
+        actual_suggestions = (feedback_models.SuggestionModel
+            .get_by_exp_id_and_status('exp_id1', 
+                                      feedback_models.STATUS_ACCEPTED))
+        
+        self.assertEqual(self.get_suggestion_models_for_test(
```

Comment:
This looks quite messy. I'd break at the first '(':

    self.assertEqual(
        self.get_suggestion_models_for_test(expected_suggestions),
        self.get_suggestion_models_for_test(actual_suggestions))